docs/user-guide/src/flake/prow.md

```diff
@@ -0,0 +1,9 @@
+# Prow flakes
+
+## Multiple PRs stuck in merge queue
```

Comment:
this https://docs.prow.k8s.io/docs/components/core/tide/maintainers/#expected-behavior-that-might-seem-strange would be very helpful reference to link somewhere in this paragraph

Reply:
Done